packages/studio-plugin/src/orchestrators/ParsingOrchestrator.ts

```diff
@@ -208,6 +215,15 @@ export default class ParsingOrchestrator {
     );
   };
 
+  private updateFilepathToDependencyTree(absPath: string) {
+    delete this.filepathToDependencyTree[absPath];
```

Comment:
should this `delete` live in `reloadFile`? in the case where a component file is deleted (HMR), I think this code wouldn't be reached?